### DISCUSSION TITLE
@truffle/workflow-compile installation

### DISCUSSION THREAD

**Discussion by VitaliiBabchenkoSecurrency** [ConsenSys-archive/truffle#5237] [Installation] [answered]
When I try to install @truffle/workflow-compile and run it I get error 
```
Error: Cannot find module '@truffle/dashboard-message-bus-client'
```

**Comment by cds-amal** [comment]
Thanks for bringing this to our attention @VitaliiBabchenkoSecurrency, and apologies for the inconvenience. Other users have raised similar dependency issues. We're working on a fix (#5230) 🤞, and plan to introduce a build step (#5236) to catch them sooner.

**Comment by cliffoo** [comment]
Hey @VitaliiBabchenkoSecurrency, [the fix](https://github.com/trufflesuite/truffle/pull/5246) should make it into this week's release. Thanks for raising this issue!

**Comment by lsqproduction** [comment, marked as answer]
The fix for this is out in [Truffle 5.5.20](https://github.com/trufflesuite/truffle/releases/tag/v5.5.20)
